### DISCUSSION TITLE
How to switch to different tabs in go-rod

### DISCUSSION THREAD

**Discussion by theiv4d** [go-rod/rod#439] [Q&A]
Hi, Is there a method to switch to different tabs on the same chromium browser? When I use MustClick() on an hyperlink element, It opens the link to another tab.

**Comment by ysmood** [comment]
#438

**Comment by gregtzar** [comment]
You can use [Browser.Pages()](https://pkg.go.dev/github.com/go-rod/rod#Browser.Pages) to get a [Pages](https://pkg.go.dev/github.com/go-rod/rod#Pages) struct which is a list of tabs and/or windows with several helpful methods attached, such as [Pages.Find()](https://pkg.go.dev/github.com/go-rod/rod#Pages.Find), [Pages.FindByURL()](https://pkg.go.dev/github.com/go-rod/rod#Pages.FindByURL), [Pages.First()](https://pkg.go.dev/github.com/go-rod/rod#Pages.First), etc. Once you get a reference to the page you want you can use [Page.Activate()](https://pkg.go.dev/github.com/go-rod/rod#Page.Activate) to focus it. In your case you can also use [Page.WaitOpen](https://pkg.go.dev/github.com/go-rod/rod#Page.WaitOpen) to grab a reference to the new window when it is launched.

**Reply by ysmood** [reply]
Nice answer! It would be great if you could append it into here: https://go-rod.github.io/#/browsers-pages?id=multiple-pages

[Contribute Doc](https://go-rod.github.io/#/contribute-doc)

**Reply by gregtzar** [reply]
@ysmood Okay I will, and I'd love to help enhance that documentation in some other places, where/how do I submit a PR for that?

**Reply by ysmood** [reply]
It's just a markdown project https://go-rod.github.io/#/contribute-doc
